pom.xml

```diff
@@ -1299,6 +1297,15 @@
                     <artifactId>maven-enforcer-plugin</artifactId>
                     <configuration>
                         <rules>
+                            <compound implementation="com.github.ferstl.maven.pomenforcers.CompoundPedanticEnforcer">
```

Comment:
Does this produce understandable, actinable error messages?

Reply:
Yes it does. It shows you the expected order and actual order with a diff. It's a really nice tool.

Reply:
```
###################################################
# COMPOUND: One does not simply write a POM file! #
###################################################

Please fix these problems:

POM_SECTION_ORDER: One does not simply write a POM file!
========================================================

Your POM has to be organized this way:

Actual Order           | Required Order
-----------------------------------------------
  modelVersion         |   modelVersion
  parent               |   parent
  groupId              |   groupId
  artifactId           |   artifactId
  version              |   version
- packaging            |
  name                 |   name
  description          |   description
                       | + packaging
  url                  |   url
  inceptionYear        |   inceptionYear
  licenses             |   licenses
  scm                  |   scm
  properties           |   properties
  modules              |   modules
  dependencyManagement |   dependencyManagement
  build                |   build
  profiles             |   profiles

```